toolset/setup/linux/languages/php.sh

```diff
@@ -49,6 +49,8 @@ printf "\n" | $IROOT/php-${VERSION}/bin/pecl -q install -f redis
 # yaf.so
 printf "\n" | $IROOT/php-${VERSION}/bin/pecl -q install -f yaf
```

Comment:
It is, but I think a couple of the PHP frameworks are built on top of it. It can probably be pulled out into its own separate install, but I don't know which frameworks that would affect.